util/getUniqueRoles.js

```diff
@@ -0,0 +1,12 @@
+export default function getUniqueRoles(teamMembers) {
```

Comment:
Ditto here:
- If a role evaluates to a truthy value, we will include that in the list of roles.
- If a role is stored as an unexpected data type, we will not include that in the list of roles.